### ISSUE TITLE
s3d pretrained model on kinetics

### ISSUE THREAD

**Issue by jwyang** [antoine77340/S3D_HowTo100M#4]
Hi, Antoine, 

thanks for open source such a great work! The pretrained weights is very useful.  Recently, I am trying compare your weights pretrained on Howto100m and those on kinetics. However, I failed to find the later one on your repo or any other places. I am wondering whether you have pretrained weights on kinetics on your side? Thanks!

best,

**Comment by antoine77340** [comment]
Unfortunately, I do not have one I am allowed to share because of the long process of open-sourcing models at DeepMind.

But I will keep you updated if we plan to release the kinetics models at some point :).

**Comment by MenghanXia** [comment]
Hi, Antoine,
Thanks for sharing your nice work. I just want know more details about the input data:
(1) As you mentioned, the input video value is: [0,1]. But I don't know whether the frames are in RGB channel order  (e.g. read by PIL) or BGR (e.g. read by cv2 / mmcv APIs) ?
(2) Should the frame pixels be normalized with ImageNet mean and std vectors?

Thanks.

Best

**Comment by antoine77340** [comment]
Hey,

(1) So the RGB channel order is the same as the PIL one
(2) No normalization is needed, the model has been trained with input image in the range [0,1].

**Comment by Hancy0369** [comment]
Hi, Antoine,

thanks for open source such a great work! The pretrained weights is very important.But due to resource constraints, I did not get weights pretrained on kinetics,I'm thinking about whether you can share the pretrained weights on kinetics on it,thanks!
This is very important to me and I look forward to your reply!

best wish!

**Comment by antoine77340** [comment]
Hi!

We unfortunately are not able to share with you the kinetics pretrained weights.
You might find another S3D implementation pre-trained on Kinetics here: https://github.com/kylemin/S3D

Best,

Antoine

**Comment by Hancy0369** [comment]
Hey！

Thank you very much for your reply. I have tried the method on the link. The weight is unreadable, but thank you very much.

Best!
